apps/web-portal/tests/_helper.mjs

```diff
@@ -0,0 +1,19 @@
+import sinon from 'sinon'
+import {MockLogger} from '@zorko-io/util-logger'
+import {ClientTypes, createClient} from '@zorko-io/util-web-api-client'
+import {WebPortalExpressApp} from '../lib'
+import {config} from '../config'
+
+export function setupAppContext() {
```

Comment:
add jsdoc with explanation of that that function do

it doesn't need a prefix `_` in the file name, just call it `helpers`

Reply:
The prefix was added according to the ava documentation. In another case, ava will try to run this file. Another solution could be setting up ava.config.json but it is in gitignore